Pester.Tests.ps1

```diff
@@ -146,6 +146,28 @@ Describe 'Style rules' {
         }
     }
 
+    It 'Pester source files lines start with a tab character' {
```

Comment:
Shouldn't this be "Pester source file lines do not start with tab character" or something along the lines of: "Indentation is done with spaces, not tabs."